package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test-disabled-till-we-add-tests": "echo \"Error: no test specified\" && exit 1",
-    "build": "find ./src -maxdepth 1 -iname '*.js' -not -name 'ampPlayer.js' -exec cat {} +> src/static/js/bundle.js && cp src/ampPlayer.js src/static/js/"
+    "build": "node build/bundle-minify.js"
```

Comment:
Isn't node a dependency too? Or is it included by default?

Reply:
it is but doesn't need to be included in the `package.json`. i'm not sure how the build agent manages platform dependencies